swagger/public_v1/template.yml

```diff
@@ -372,6 +372,47 @@ components:
         version:
           type: string
           example: "1.0"
+    LocationAttributes:
+      type: object
+      properties:
+        code:
+          type: string
+          example: "W"
+        location_name:
+          type: string
+          example: "Henry Maynard Primary School"
+        address1:
+          type: string
+          example: "Maynard Road"
+        address2:
+          type: string
+          example: ""
+        address3:
+          type: string
+          example: ""
+        address4:
+          type: string
+          example: "London"
```

Comment:
For all these address properties, we use the wording "Building and street" for `address1` and `address2`, and then "Town or city" for `address3` and "County" for for `address4`. In the provider attributes I've tried to use language that's closer to that ... I think we should do that here.

See [the provider attributes in my PR](https://github.com/DFE-Digital/teacher-training-api/pull/1278/files#diff-9d39c599c09a32142f10c55337b8a56eR78) and [@gpeng's comments](https://github.com/DFE-Digital/teacher-training-api/pull/1278#discussion_r400029777).

Reply:
I've made these changes and it should now match your PR